source/_lovelace/alarm-panel.markdown

```diff
@@ -12,14 +12,11 @@ The Alarm Panel card allows you to arm and disarm your [alarm control panel](/in
 Screenshot of the Alarm Panel card.
 </p>
 
-To add the Alarm Panel card to your user interface, click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Click the "Add Card" button in the bottom right corner and select **Alarm Panel** from the card picker. All options for this card can be configured via the user interface.
+All options for this card can be configured via the user interface.
```

Comment:
Why change this?

Reply:
Resolving this as we have the discussion below